ui/shared/DebounceTimer.qml

```diff
@@ -0,0 +1,10 @@
+import QtQml 2.13
+
+Timer {
```

Comment:
We already have a Debounce thing: https://github.com/status-im/nim-status-client/tree/master/ui/imports/Backpressure
Maybe it's not exactly what you're doing here, but I'm pointing it out in case there is some overlap

Reply:
I noticed that today haha. It looks like it's the same thing. I'll swap it over.